package.json

```diff
@@ -12,28 +12,35 @@
   },
   "scripts": {
     "build": "gulp",
-    "create-push-notifications-service-user": "node dist/tools/createPushNotificationsServiceUser.js",
-    "create_db": "sequelize db:create",
-    "create_missing_users": "node dist/tools/create_missing_users.js",
-    "delete_db": "sequelize db:drop",
-    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
+    "create_push_notifications_service_user": "babel-node tools/createPushNotificationsServiceUser.js",
```

Comment:
Again, more curiosity than probing...just wondering how and why we're using `babel-node` here.

Reply:
`babel-node` allows ES6-8 language features that wouldn't be available in node 8.14.0 (the node version specified in `package.json:engines`), e.g. import vs require. 

That said, it doesn't look like `tools/createPushNotificationsServiceUser.js` proper requires language features that aren't available in node 8, so either `babel-node` or `node` should work.

Reply:
Ah yup you're right! I just converted anything that previously ran `node` on something out of the `dist/` folder to use `babel-node` instead (the assumption being that if it had to compiled before, it would probably need babel-node), but this script in particular does not need babel-node.